scripts/og-image/template.html

```
@@ -0,0 +1,141 @@
+<!--
+  Source of truth for the 1200x630 link-preview card rendered to
+  src/ludamus/static/og-image.jpg by build.mjs.
+
+  The scrim knobs below are the same custom properties the tweaking widget
+  exposes, so values dialled in there paste straight back into :root.
+-->
+<meta charset="utf-8">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)